src/geo/map/torque-layer.js

```diff
@@ -110,36 +126,35 @@ var TorqueLayer = LayerModelBase.extend({
     this.set('renderRange', {});
   },
 
-  isEqual: function(other) {
+  isEqual: function (other) {
     var properties = ['query', 'query_wrapper', 'cartocss'];
     var self = this;
-    return this.get('type') === other.get('type') && _.every(properties, function(p) {
-      return other.get(p) === self.get(p);
-    });
+    return this.get('type') === other.get('type') && _.every(properties, function (p) {
+        return other.get(p) === self.get(p);
+      });
   },
 
-  isVisible: function() {
+  isVisible: function () {
```

Comment:
If we're fixing all linter errors from this file... let's remove it from .eslintignore!